app/forms/add_to_order_form.rb

```diff
@@ -25,7 +25,28 @@ def initialize(original_order)
   def save
     raise(ActiveRecord::RecordInvalid, self) unless valid?
 
-    add_to_order!
+    @facility_id = @facility_id&.to_i
+
+    order_project = Projects::Project.find_by(
+      name: "#{current_facility.abbreviation}-#{original_order.id}",
+      facility_id: current_facility.id
+    )
+
+    return add_to_order!(order_project) if @original_order.facility.id == @facility_id
+
+    if order_project.nil?
+      create_cross_core_project_and_add_order!
+    else
+      facility_order_in_project = order_project.order_details.find { |od| od.order.facility_id == @facility_id }&.order
```

Comment:
this will need to change in the next PR, right?

Reply:
I don't think so. What's not right with this?